AppController/djinn.rb

```diff
@@ -3512,6 +3523,7 @@ def start_api_services()
     if my_node.is_load_balancer?
       threads << Thread.new {
         start_ejabberd()
+        configure_tq_haproxy()
```

Comment:
Should we wait until we implement some sort of authentication before enabling it by default?